src/responder/pam/pamsrv.h

```diff
@@ -160,7 +160,7 @@ errno_t filter_responses(struct pam_ctx *pctx,
                          struct response_data *resp_list,
                          struct pam_data *pd);
 
-errno_t pam_get_auth_types(struct pam_data *pd,
+errno_t pam_get_auth_types(struct pam_data *pd, bool only_return_auth_types,
```

Comment:
`pam_get_auth_types` vs `only_return_auth_types` is confusing naming.
Not sure if something like 'skip_cert_info' would be better, but warrants a comment at least, imo.

Reply:
Hi,

instead of renaming I split the functionality into two different functions.

bye,
Sumit